src/abft-participants.md

```diff
@@ -0,0 +1,55 @@
+# Identity, Authorization, and Authentication
+
+A _player_ is uniquely identified by a 256-bit string \\(I\\) called an
+_address_.
+
+Each player owns exactly one _participation keypair_. A participation
+keypair consists of a _public key_ \\(pk\\) and a _secret key_ \\(sk\\).
+
+A keypair is an opaque object defined in the [specification
```

Comment:
```suggestion
A keypair is defined in the [specification
```
I don't understand what is conveyed by calling them an "opaque object".  We define them, so I don't see how they are "opaque".